GitUI/Editor/FileViewer.cs

```diff
@@ -811,21 +811,25 @@ private Task ViewItemAsync(string fileName, bool isSubmodule, Func<Image> getIma
                 return _async.LoadAsync(getImage,
                             image =>
                             {
+                                if (image == null)
+                                {
+                                    ResetForText(null);
+                                    internalFileViewer.SetText($"Cannot view image {fileName}", openWithDifftool);
```

Comment:
Localise please

Reply:
Done
Generally I believe translation is overkill for something like this, but I allow it as it is (at least currently) shown for removed images.
Unless there are further comments in the next two days, I will squash and schedule a merge